regexes/device/mobiles.yml

```diff
@@ -7399,6 +7525,10 @@ Timovi:
   regex: 'Tmovi'
   device: 'smartphone'
   models:
+    - regex: 'Prime'
```

Comment:
Might be better to use `Tmovi[ _]Prime`, so it also matches `Mozilla/5.0 (Linux; U; Android 4.4.2; es-LA; Tmovi_Prime Build/KOT49H) AppleWebKit/528.5+ (KHTML, like Gecko) Version/3.1.2 Mobile Safari/525.20.1 UCBrowser/9.7.0.520 Mobile`